### PR TITLE
#84 Update the tutorial for parathread deployment.

### LINKED ISSUE #84: 🎁 [Feature Request]: Test and add guidelines about connecting to relay chain
### What is the feature you would like to see?

We have to test whether our template works well with relay chains (Kusama or Polkadot). 
After verifying everything is working, we should enhance our documentation with:

* Guide on how to run our template and connect to relay chain (Kusama or Polkadot)

### Contribution Guidelines

- [X] I agree to follow this project's Contribution Guidelines

### PR DESCRIPTION
**Pull request by KitHat**
This is an update for tutorial that reflects actions that were done to set up a parathread against Rococo.

Closes #84 

#### PR Checklist

- [x] Documentation

### CODE REVIEW

**Review by ozgunozerk — APPROVED** [review]
Amazing work, thanks Nikita!

File: docs/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -9,38 +9,108 @@ IMPORTANT: This repo contains code in its early stages. Please be aware that it
...
+    --bob \
+    --collator \
+    --force-authoring \
+    --chain raw-parachain-chainspec.json \
+    --base-path <path to datadir> \
+    --port 40333 \
+    --rpc-port 8845 \
+    -- \
+    --execution wasm \
+    --chain <path to the Rococo chainspec> \
```
[the remark this answers is not in the document]
Comment (KitHat) [reply docs/modules/ROOT/pages/index.adoc:80]: I can do it, I agree, it would look prettier in two boxes.

**Comment by KitHat** [comment]
> Should this tutorial to connect to rococo go in a separate page instead of on the main docs page?

@4meta5 it can go to a separate page, but I was inspired with an index page of Cairo Contracts documentation: https://docs.openzeppelin.com/contracts-cairo/0.8.1/

I think that it is a good idea for the first page to guide a user straight to a result and then present what they can change.

**Comment by KitHat** [comment]
I will figure out with Joao if this guide would fulfill the Parity

**Pull request merged**